doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -14,9 +14,15 @@ GNU ARM Embedded
 
    .. warning::
 
-      Like the Zephyr repository, do not install the toolchain in a directory
-      with spaces anywhere in the path. On Windows, we'll assume you install
-      into the directory :file:`C:\\gnu_arm_embedded`.
+      Do not install the toolchain into a path with spaces. On
+      Windows, we'll assume you install into the directory
+      :file:`C:\\gnu_arm_embedded`.
+
+   .. warning::
+
+      The latest release of the Windows toolchain has a critical bug,
```

Comment:
Please do not use `the latest release`. Instead specify the exact release that has the issue, otherwise this will quickly become untrue.

Reply:
I figured people would be too lazy to string compare the version, but would not be too lazy to read this and understand that they can't just download the latest version.

Also, when there exists a new version, I expect that the issue will be fixed, so we will need a documentation update anyway.

What about a compromise, something like:

> The latest release of the Windows toolchain (8-2018-q4-major) has a critical bug,

Reply:
That's probably good enough.

Reply:
I agree with the compromise then.

Reply:
Actually, thinking back at this. Why not specify the revision and remove `latest version` and keep this in the doc for the next year or so so that users that have the one that has an issue can see that this is a known issue?

Reply:
Because I figured people would be too lazy to string compare the version, but would not be too lazy to read this and understand that they can't just download the latest version.

Also, when there exists a new version, I expect that the issue will be fixed, so we can safely remove this warning and expect users to use the latest version.

Reply:
revision is now specified, and latest version has been removed.

Reply:
@carlescufi : Your review comment has been addressed, please re-asses the review.

Reply:
@carlescufi : Ping